packages/minifier-js/package.js

```diff
@@ -8,7 +8,15 @@ Npm.depends({
 });
 
 Package.onUse(function (api) {
+  api.use('ecmascript');
   api.use('babel-compiler');
```

Comment:
Shouldn't this be a dependency of `standard-minifier-js` instead?

Reply:
We could potentially get it in through imply. But for now I would leave it and rather have a change like this in a separate PR. This one is big enough and has already gone on for long enough.

Reply:
@zodern that is unless you find it absolutely necessary to address right now.

Reply:
Does `minifier-js` use `babel-compiler`? If not, it would be a two line change: (re-adding [this line](https://github.com/meteor/meteor/pull/10967/commits/76c21f872906e87dd5e276a3ca53bdcb05a25031#diff-9353693b93ed9cf0b487fabfc08ea75916030f1bd7afe25373192f488e31550bL12)) and removing `babel-compiler` from minifer-js.

Fixing in a separate PR is fine.